Simplified/AxisService/AxisService.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  AxisService.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-01.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+@objc
+protocol BookDownloadBroadcasting {
+  func failDownloadWithAlert(forBook book: NYPLBook)
+  func replaceBook(_ book: NYPLBook,
+                   withFileAtURL sourceLocation: URL,
+                   forDownloadTask downloadtask: URLSessionDownloadTask) -> Bool
+}
```

Comment:
These methods are already present in NYPLMyBooksDownloadCenter so new changes need to be introduced.